grpc/src/main/java/com/linecorp/armeria/server/grpc/FramedGrpcService.java

```diff
@@ -327,6 +330,8 @@ private <I, O> AbstractServerCall<I, O> newServerCall(
             HttpResponse res, @Nullable CompletableFuture<HttpResponse> resFuture,
             SerializationFormat serializationFormat, @Nullable Executor blockingExecutor) {
         final MethodDescriptor<I, O> methodDescriptor = methodDef.getMethodDescriptor();
+        final GrpcExceptionHandlerFunction grpcExceptionHandler = registry.wrapGrpcExceptionHandler(
+                methodDef, grpcExceptionHandlerFunction);
```

Comment:
Should we early create the final `GrpcExceptionHandlerFunction` when `GrpcService` is created for efficiency? Creating a new `GrpcExceptionHandlerFunction` for every call would cause additional GC pressure.